shell/platform/windows/settings_plugin_win32.h

```diff
@@ -39,11 +39,15 @@ class SettingsPluginWin32 : public SettingsPlugin {
 
  private:
   void WatchPreferredBrightnessChanged();
+  void WatchTextScaleFactorChanged();
 
   HKEY preferred_brightness_reg_hkey_ = NULL;
+  HKEY text_scale_factor_reg_hkey_ = NULL;
 
   std::unique_ptr<EventWatcherWin32> preferred_brightness_changed_watcher_{
       nullptr};
+  std::unique_ptr<EventWatcherWin32> text_scale_factor_changed_watcher_{
+      nullptr};
```

Comment:
nit: There's no need to explicitly initialize `unique_ptr` ;)

Reply:
done